saleor/order/migrations/0081_merge_20200309_0952.py

```diff
@@ -0,0 +1,13 @@
+# Generated by Django 2.2.10 on 2020-03-09 14:52
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
```

Comment:
Instead of merging migration, you could just remove your migration and re-generate it one more time.